infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -1108,10 +1124,8 @@ public void reset() {
             }
         });
 
-        serverContext.setStartingAddress(0L);
-        serverContext.setTailSegment(0L);
-        startingAddress = 0L;
-        lastSegment = 0L;
+        dataStore.resetStartingAddress();
```

Comment:
Why don't you just delete and recreate the log dir?

Reply:
will do it in next PR